assets/scss/components/_dropdown.scss

```
@@ -0,0 +1,58 @@
+$dropdown-offset-right: $length-6;
+
+details.dropdown {
+    position: relative;
+
+    summary {
+        appearance: none;
+    }
+
+    &[open] {
+        z-index: 10;
+    }
+
+    &[open] summary::before {
+        content: '';
+        position: fixed;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+    }
+
+    .dropdown-content {
+        position: absolute;
+        top: $length-32;
+        right: $dropdown-offset-right;
+
+        margin: 0;
+        padding: 0;
+
+        z-index: 21;
+
+        @include shadow-4;
+
+        &::before,
+        &::after {
+            content: " ";
+            position: absolute;
+            display: block;
+            width: 0;
+            height: 0;
+        }
+        &::before {
+            top: -10px;
+            right: #{$length-14 - $dropdown-offset-right};
```

Petit retour en arrière, ne pas mettre de variable étant donné que c'est pour la flèche donc c'est du pixel-perfect.
```suggestion
            right: #{14px - $dropdown-offset-right};
```